react/utils/routes.ts

```diff
@@ -211,10 +211,16 @@ const promiseWithCounterWrapper = <T = any>(
     })
 }
 
-export const isPrefetchActive = () =>
+export const isPrefetchEnabled = (
+  storeSettings: Record<string, any> | null | undefined
+) => isEnabled('PREFETCH') && storeSettings?.enablePrefetch
```

Comment:
It is weird when a `isSomething()` function returns `undefined`.

Reply:
do you even javascript bruh?